packages/prism-theme-odyssey/src/utils/CustomPropertyInspector.js

```diff
@@ -0,0 +1,122 @@
+const Storage = window.localStorage;
+
+class CustomPropertyInspector {
+  constructor({propertiesElement, inspectorElement}) {
+    this.properties = this.setCustomProperties(propertiesElement);
```

Comment:
consider removing internal state by just passing this value directly into `renderInspector` without assignment to `this`.